### PR TITLE
Fix JMS and WebSocket Examples 

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
Fixes JMS and WebSocket examples by applying new syntax changes

### CODE REVIEW

File: examples/send-jms-message-to-http-endpoint/send_jms_message_to_http_endpoint.bal [hunk 1]
```diff
@@ -3,81 +3,77 @@ import ballerina/jms;
...
 // Backend service that receive the forwarded message from the broker.
-service<http:Service> backend bind { port: 9090 } {
+service backend on new http:Listener(9090) {
 
     @http:ResourceConfig {
         methods: ["POST"],
         path: "/jms"
     }
-    jmsPayloadReceiver(endpoint conn, http:Request req) {
+    resource function jmsPayloadReceiver(http:Caller conn, http:Request req) {
```
Comment (chamil321) [inline examples/send-jms-message-to-http-endpoint/send_jms_message_to_http_endpoint.bal]: Shall we rename `conn` to `caller`?

File: examples/send-jms-message-to-http-endpoint/send_jms_message_to_http_endpoint.bal [hunk 2]
```diff
@@ -3,81 +3,77 @@ import ballerina/jms;
...
-
-        match (message.getTextMessageContent()) {
-            string textContent => {
-                log:printInfo("Message received from broker. Payload: "
-                              + textContent);
-
-                forwardToBakend(untaint textContent);
-            }
-            error e => log:printError("Error while reading message", err=e);
+    resou rce function onMessage(jms:QueueReceiver consumer,
```
Comment (chamil321) [inline examples/send-jms-message-to-http-endpoint/send_jms_message_to_http_endpoint.bal]: resource*

File: examples/jms-synchronous-queue-message-receiver/jms_synchronous_queue_message_receiver.bal [hunk 3]
```diff
@@ -14,32 +14,26 @@ jms:Session jmsSession = new(jmsConnection, {
...
+    if (result is jms:Message) {
+        var messageText = result.getTextMessageContent();
+        if (messageText is string) {
+            log:printInfo("Message : " + messageText);
+        } else {
+            log:printError("Error occurred while reading"
+                         + "message", err=messageText);
         }
+    } else {
+            log:printInfo("Error receiving message. " + err=result);
```
Comment (chamil321) [inline examples/jms-synchronous-queue-message-receiver/jms_synchronous_queue_message_receiver.bal]: indentation issue? 
`err=result` -> `err = result`

File: examples/http-to-websocket-upgrade/http_to_websocket_upgrade.bal [hunk 4]
```diff
@@ -46,25 +46,25 @@ service<http:Service> httpService bind { port: 9090 } {
 // - If  WebSocketServiceConfig is defined without the path, sub protocols, idle timeout etc... can be configured.
 // - If path is defined in the WebSocketServiceConfig it shall be ignored.
 // - This service can also be bound to a different endpoint in which case the path configuration becomes useful.
-@http:WebSocketServiceConfig {
-    subProtocols: ["xml, json"],
-    idleTimeoutInSeconds: 20
-}
-service<http:WebSocketService> wsService {
+//@http:WebSocketServiceConfig {
```
Comment (wggihan) [inline examples/http-to-websocket-upgrade/http_to_websocket_upgrade.bal]: Shall we remove these commented lines if we don't want?

File: examples/http-to-websocket-upgrade/http_to_websocket_upgrade.bal [hunk 5]
```diff
@@ -46,25 +46,25 @@ service<http:Service> httpService bind { port: 9090 } {
...
+//
+//}
+service wsService = @http:WebSocketServiceConfig {subProtocols: ["xml, json"],
+                                   idleTimeoutInSeconds: 20} service {
```
Comment (riyafa) [inline examples/http-to-websocket-upgrade/http_to_websocket_upgrade.bal]: Shall we format this properly?

File: examples/transactional-queue-message-producer/transactional_queue_message_producer.bal [hunk 6]
```diff
@@ -13,28 +13,25 @@ jms:Session jmsSession = new(jmsConnection, {
...
+jms:QueueSender queueSender = new({
     session:jmsSession,
     queueName:"MyQueue"
-};
+});
 
 public function main() {
     // Message is published within the transaction block.
     transaction {
     // Create a Text message.
```
Comment (samgnaniah) [inline examples/transactional-queue-message-producer/transactional_queue_message_producer.bal]: Shall we make "Text" lowercase here?

File: tests/ballerina-integration-test/src/test/resources/jms/jmsservices/03_jms_map_message_subscriber.bal [hunk 7]
```diff
@@ -33,9 +33,6 @@ service jmsListener2 on topicSubscriber2 {
              io:print(messageRetrieved["c"]);
              io:println(messageRetrieved["d"]);
              var retrievedBlob = <byte[]>messageRetrieved["e"];
```
Comment (VijithaEkanayake) [inline tests/ballerina-integration-test/src/test/resources/jms/jmsservices/03_jms_map_message_subscriber.bal]: Shall we assign this to byte[]?

```suggestion
             byte[] retrievedByteArray = <byte[]>messageRetrieved["e"];
```

File: examples/jms-synchronous-queue-message-receiver/jms_synchronous_queue_message_receiver.bal (hunk 3 above)
Comment (VijithaEkanayake) [inline examples/jms-synchronous-queue-message-receiver/jms_synchronous_queue_message_receiver.bal]:
```suggestion
            log:printInfo("Error receiving message. ", err = result);
```

**Pull request merged**